### PR TITLE
Integrated drone, made linter happy and added standard files

### PR DESCRIPTION
**Pull request by tboerger**

### CODE REVIEW

**Comment by strk** [comment]
trust-based LGTM (didn't really look)

File: cache/mock/cache.go [hunk 1]
```diff
@@ -1,7 +1,7 @@
 package mocks
 
-import "github.com/go-gitea/lgtm/cache"
-import "github.com/stretchr/testify/mock"
+import cache "github.com/go-gitea/lgtm/cache"
```
Comment (metalmatze) [inline cache/mock/cache.go:3]: Why do we need _cache_ and _mock_ explicitly upfront in this import?

**Comment by tboerger** [comment]
OK, now it's building correctly and IMHO safe to merge.

**Review by metalmatze — CHANGES_REQUESTED** [review]

File: cache/mock/cache.go (hunk 1 above)
Comment (tboerger) [reply cache/mock/cache.go:3]: This is automatically generated by mockify.

**Comment by andreynering** [comment]
Also trust-based LGTM

File: cache/mock/cache.go (hunk 1 above)
Comment (metalmatze) [reply cache/mock/cache.go:3]: Ok. Thanks. This doesn't break anything. So I'm going to merge.
Really good job 👍

**Comment by tboerger** [comment]
Thanks guys!

**Pull request merged**